apps/src/javalab/JavalabEditor.jsx

```diff
@@ -360,6 +362,40 @@ class JavalabEditor extends React.Component {
     });
   }
 
+  // This moves the active tab to the left in the tab menu
+  moveTabLeft() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
+    let index = newTabs.indexOf(activeTabKey);
+    if (index > 0) {
+      this.swapTabsSetKeyCloseTab(index - 1, index, newTabs);
+    }
+  }
+
+  // This moves the active tab to the right in the tab menu
+  moveTabRight() {
+    const {activeTabKey, orderedTabKeys} = this.props;
+    let newTabs = [...orderedTabKeys];
+    let index = newTabs.indexOf(activeTabKey);
+    if (index < newTabs.length - 1) {
+      this.swapTabsSetKeyCloseTab(index, index + 1, newTabs);
+    }
+  }
+
+  swapTabsSetKeyCloseTab(index1, index2, newTabs) {
```

Comment:
Might be worth adding a test to `JavalabEditorTest` to test out this functionality too (making sure indices swap correctly, or don't swap if out of bounds, making sure menu closes, etc)

Reply:
Good idea - added a handful of tests for this.